css-flexbox/css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100%/6);
```

Comment:
You don't want the `#card-deck` to be one sixth of the available width, it's meant to hold all of the pokemon cards. This should be applied to the `.card-container` rule